picard/ui/options/plugins.py

```diff
@@ -107,7 +110,7 @@ def save_state(self):
         config.persist["plugins_list_sort_section"] = header.sortIndicatorSection()
         config.persist["plugins_list_sort_order"] = header.sortIndicatorOrder()
         try:
-            selected = self.items[self.ui.plugins.selectedItems()[0]].module_name
+            selected = self.items[self.ui.plugins.selectedItems()[COLUMN_NAME]].module_name
```

Comment:
`if i.checkState(0) == QtCore.Qt.Checked:` at line 208 should use COLUMN_NAME too i think.